openedx/features/course_experience/templates/course_experience/course-sock-fragment.html

```diff
@@ -0,0 +1,70 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+
+<%block name="content">
+    % if show_course_sock:
+    <div class="verification-sock">
+        <button type="button" class="btn btn-brand focusable action-toggle-verification-sock">
+            ${_("Learn About Verified Certificate")}
+        </button>
+        <div class="verification-main-panel">
+            <div class="verification-desc-panel content-main">
+                <h2>${_("edX Verified Certificate")}</h2>
+                <h4>${_("Why upgrade?")}</h4>
+                <ul>
+                    <li>${_("Official Proof of completion")}</li>
+                    <li>${_("Easily shareable certificate")}</li>
+                    <li>${_("Proven motivator to complete the course")}</li>
+                    <li>${_("Certificate purchases help edX continue to offer free courses")}</li>
+                </ul>
+                <h4>${_("How it works")}</h4>
+                <ul>
+                    <li>${_("Choose the course that's right for you")}</li>
+                    <li>${_("Pay the Verified Certificate upgrade fee")}</li>
+                    <li>${_("Verify your identity with a webcam and government-issued ID")}</li>
+                    <li>${_("Study hard and pass the course")}</li>
+                    <li>${_("Share your certificate with friends, employers, and others")}</li>
+                </ul>
+                <h4>${_("edX Learner Stories")}</h4>
+                <div class="learner-story-container">
+                    <img class="student-image" alt="Student Image" src="${static.url('images/logo.png')}" />
+                    <div class="story-quote">
+                        ${_("My certificate motivated me to work my hardest and try my best, so I can "
```

Comment:
I'm struggling with whether we should be localizing all of this marketing material. I'm not convinced it will make sense if each individual string is translated independently. I'm also not at all sure we should be localizing quotes from people, because then it is no longer their quote. I think this will be confusing when the translators receive all these strings to translate.

Having said this, it doesn't make sense to have a purely English language sock either, unless we hide it for all other languages. @marcotuts, what do you think?